src/keymaster-api.js

```diff
@@ -255,26 +273,32 @@ v1router.post('/response/verify', async (req, res) => {
         const verify = await keymaster.verifyResponse(response, challenge);
         res.json(verify);
     } catch (error) {
-        res.status(400).send(error.toString());
+        res.status(400).send({ error: error.toString() });
     }
 });
 
 v1router.post('/groups', async (req, res) => {
     try {
         const { name, registry } = req.body;
+        if (!name || !registry) {
+            return res.status(400).send({ error: 'Name and registry are required' });
+        }
```

Comment:
Same question as above. Also, not specifying the registry is valid, the default registry will be used.

Reply:
redundant code removed